win32ss/gdi/ntgdi/freetype.c

```diff
@@ -719,8 +719,8 @@ VOID FASTCALL IntWidthMatrix(FT_Face face, FT_Matrix *pmat, LONG lfWidth)
 VOID FASTCALL IntEscapeMatrix(FT_Matrix *pmat, LONG lfEscapement)
 {
     FT_Vector vecAngle;
-    /* Convert from angle in tenths of degrees to 'FT_Angle' degrees */
-    FT_Angle angle = FT_FixedFromFloat((FLOAT)lfEscapement / 10);
+    /* Convert the angle in tenths of degrees into degrees as a 16.16 fixed-point value */
+    FT_Angle angle = lfEscapement * (1 << 16) / 10;
```

Comment:
By the way do we have some sort of macro that has a human-understanding name like "INT_TO_FIXED" or some sort of thing, that does this shift automatically?

Reply:
`freetype\internal\ftcalc.h` has `INT_TO_FIXED` macro as follows:
```c
#define INT_TO_FIXED( x )      ( (FT_Long)(x) * 65536 )  /* << 16 */
```

Reply:
Done.

Reply:
Wow I'm surprised that I guessed correctly the actual name of the macro xD